plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineSdkClasspathContainerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.junit.Test;
+
+public class AppEngineSdkClasspathContainerTest {
+
+  @Test
+  public void testGetPath() {
+    IPath path = new AppEngineSdkClasspathContainer().getPath();
+    assertThat(path.segmentCount(), is(1));
```

Comment:
I thought you could say something like 
assertThat(path.segmentCount()).is(1)?
If not, I think assertEquals is clearer